packages/forms/src/UIForm/fields/Select/Select.component.js

```diff
@@ -30,13 +39,14 @@ export default function Select({ id, isValid, errorMessage, onChange, onFinish,
 			label={title}
 			labelAfter
 			required={schema.required}
+			valueIsUpdating={valueIsUpdating}
```

Comment:
replacing by 'updatingValue' would have the same meaning

Reply:
`would have the same meaning` --> So let's not change it :D